arch/arm/src/armv7-r/arm_syscall.c

```diff
@@ -526,10 +533,18 @@ uint32_t *arm_syscall(uint32_t *regs)
         break;
     }
 
+  regs = (uint32_t *)CURRENT_REGS;
```

Comment:
do we beed to add `group_addrenv(NULL);` in prior to this operation?

Reply:
Don't need since armv7-r like armv7-m doesn't have MMU, but MPU.

Reply:
Then let's remove `#include "addrenv.h"` from this file

Reply:
ARCH_KERNEL_STACKSIZE depends by #include "addrenv.h"